dji_sdk/global_position target
---
bool success